lms/djangoapps/organizations/views.py

```diff
@@ -115,7 +115,7 @@ def users(self, request, pk):
                         enrollments = CourseEnrollment.enrollments_for_user(user).count()
                         user_data['course_count'] = enrollments
                     if str2bool(include_grades) and course_key:
-                        user_grades = {'grade': 0, 'proforma_grade': 0}
+                        user_grades = {'grade': 0, 'proforma_grade': 0, 'complete_status': False}
```

Comment:
Shouldn't always be returning false, should be defaulting to false in case there is no gradebook record.